packages/types/src/interfaces/injects.ts

```diff
@@ -52,6 +53,9 @@ export const typesBundle: OverrideBundleType = {
         {
           minmax: [37, undefined],
           types: {
+            // In our staking runtime module, we use StakingLedgerTo223 definition of polkadot's staking ledger
+            StakingLedger: 'StakingLedgerTo223',
```

Comment:
in this version of master may be we don't have StakingLedgerTo223
as master is still using 2.7.1

Reply:
ahh true think this snuck over from develop branch when I cherry picked, will remove